tests/robot-tests/tests/admin_and_public/bau/publish_methodology.robot

```diff
@@ -93,6 +93,14 @@ Verify that the methodology is publicly accessible
     user waits until h1 is visible    ${PUBLICATION_NAME}
     user waits until page contains title caption    Methodology
 
+Verify that the methodology displays a link to the publication
+    ${relatedInformation}=    get webelement    css:[aria-labelledby="related-information"]
```

Comment:
We've had a few intermittent StaleElementException failures in the tests and, discussing this with Dunc and Luke, I think they can happen in situations like this, where you're grabbing the element immediately after a page has loaded, because React sometimes rerenders the page - I think this happens with the Admin dashboard when loading the theme/topic dropdowns. Even though its inefficient, it might be better to reuse the selector each time.

Not saying you need to change anything here, but just something to bear in mind.

Reply:
Thanks, I took on your suggestion in case it reduces the chances of seeing StaleElementException